src/Database/Postgres.purs

```diff
@@ -90,93 +85,87 @@ connectionInfoFromConfig c p = unsafeCoerce
   }
 
 -- | Makes a connection to the database via a Client.
-connect :: forall eff. Pool -> Aff (db :: DB | eff) Client
-connect = fromEffFnAff <<< connect'
+connect :: Pool -> Aff Client
+connect = fromEffectFnAff <<< connect'
 
 -- | Runs a query and returns nothing.
-execute :: forall eff a. Query a -> Array SqlValue -> Client -> Aff (db :: DB | eff) Unit
-execute (Query sql) params client = void $ fromEffFnAff $ runQuery sql params client
+execute :: forall a. Query a -> Array SqlValue -> Client -> Aff Unit
+execute (Query sql) params client = void $ fromEffectFnAff $ runQuery sql params client
 
 -- | Runs a query and returns nothing
-execute_ :: forall eff a. Query a -> Client -> Aff (db :: DB | eff) Unit
-execute_ (Query sql) client = void $ fromEffFnAff $ runQuery_ sql client
+execute_ :: forall a. Query a -> Client -> Aff Unit
+execute_ (Query sql) client = void $ fromEffectFnAff $ runQuery_ sql client
 
 -- | Runs a query and returns all results.
-query :: forall eff a
-  . Decode a
-  => Query a -> Array SqlValue -> Client -> Aff (db :: DB | eff) (Array a)
-query (Query sql) params client = do
-  rows <- fromEffFnAff $ runQuery sql params client
-  either liftError pure (runExcept (sequence $ decode <$> rows))
+query :: forall b a
+  . (b -> String) -> (Foreign -> Either b a) -> Query a -> Array SqlValue -> Client -> Aff (Array a)
+query print decode (Query sql) params client = do
+  rows <- fromEffectFnAff $ runQuery sql params client
+  either (liftError print) pure (sequence $ decode <$> rows)
 
 -- | Just like `query` but does not make any param replacement
-query_ :: forall eff a
-  . Decode a
-  => Query a -> Client -> Aff (db :: DB | eff) (Array a)
-query_ (Query sql) client = do
-  rows <- fromEffFnAff $ runQuery_ sql client
-  either liftError pure (runExcept (sequence $ decode <$> rows))
+query_ :: forall b a
+  . (b -> String) -> (Foreign -> Either b a) ->  Query a -> Client -> Aff (Array a)
```

Comment:
There is an extra space before the `Query a` here and also in the next few type defs.